### DISCUSSION TITLE
Stat Logging plugin

### DISCUSSION THREAD

**Discussion by PH5HP** [e107inc/e107#5234] [General]
Hi,

Where can I find the Stats Logging plugin? I searched in the plugin section (installed and not installed) and also in the Find Plugins option. I tried the internet, but I can not find this plugin.

Anyone?

Kind regards,
Henk - www.ph5hp.nl - https://fileserver.ph5hp.nl
![image](https://github.com/e107inc/e107/assets/140949730/9c60f1cf-f6ba-428b-81d4-34532408620a)

**Comment by tgtje** [comment]
@PH5HP  [here](https://github.com/e107inc/log) 
it is no longer part of the system.

**Reply by PH5HP** [reply]
Hi,

TNX for the reply. This plugin does not work in my E107. No error, but nothing happened and I can not go to the settings to configure it. I do not understand that they left out this in E107 V2 and keeping the menu in.  Very strange.

Kind regards,
Henk - www.ph5hp.nl - https://fileserver.ph5hp.nl

**Reply by Jimmi08** [reply]
In PHP 7.4 it works normally (after activation in settings, users are displayed on dashboard)
Edit: switched to PHP 8.1, no visible errors on wampp, latest github is used

**Reply by Alex-e107nl** [reply]
Every server has a kind of stats configured, i think that was the purpose.. look at [https://github.com/e107inc/awstats](https://github.com/e107inc/awstats) but i cannot get it to work on my server... Asked for instructions a while ago but... nothing yet
